### DISCUSSION TITLE
Installation error

### DISCUSSION THREAD

**Discussion by Truleycopf** [IfcOpenShell/IfcOpenShell#4798] [Q&A]
Hi all. I try install ifcopenshell but I have problems about VS's packages. Which are packages I need to install from Visual Studio installer? Or How I to solve this problem?
![изображение_2024-06-04_143109143](https://github.com/IfcOpenShell/IfcOpenShell/assets/171221158/53f5285a-591e-4794-9a4a-acea515741df)
![image](https://github.com/IfcOpenShell/IfcOpenShell/assets/171221158/4dd67d7d-6d08-4c58-8f90-fcbd590233dc)

**Comment by Martin15135215** [comment]
See https://github.com/IfcOpenShell/IfcOpenShell/discussions/4712#discussioncomment-9520217, maybe it could help

**Reply by Truleycopf** [reply]
Thanks. I saw it and had same problem but after that I installed "VS Installer (Build tools)" I got this problems. I think it's depend of installed packages from "VS Installer (Build tools)". May be somebody knows which libraries should install from "VS Installer (Build tools)"?

**Reply by Martin15135215** [reply]
I installed IfcOpenShell-Python with Mamba/Conda, and there I did not get this error. Unfortunately, I don't know how to solve this problem.
